### DISCUSSION TITLE
HBW-LC-RGBWW-IN6-DR: Nummer Entitäts-Domain als Licht Entitäts-Domain

### DISCUSSION THREAD

**Discussion by FlorianWicke** [SukramJ/aiohomematic#851] [General]
Hallo zusammen,

ich habe folgenden Aufbau:

CCU3
RS485 Bus-Gateway
HomeAssistent auf einem Raspi3 (erstmal zum testen).
RGB Dimmer von einem Dritthersteller. (Haus-Bus.de)

Jetzt zu meinen Problem. Diesen RGB Dimmer verwende ich als 6-Fach Dimmer und habe CCT-LED Stripes angeschlossen. 
Dieser Dimmer wird mir in Home-Assistant als Number-Entität weitergegeben. Somit ist es mir nur möglich die Fartemparatur über Zahlen zu steuern. Über die Licht-Entitäts-Domain ist es jedoch möglich die Farbtemperatur über einen Regler zu steuern.

Gibt es eine Möglichkeit dieses Gerät der Licht Entitäts-Domain zuzuführen.
Es würde die Arbeit dann mit adaptiven Licht (was das eigentliche Ziel ist) erleichtern.

Vielen Dank im voraus!
Lg
Flo

**Comment by SukramJ** [comment]
> Gibt es eine Möglichkeit dieses Gerät der Licht Entitäts-Domain zuzuführen.

Das sollte möglich sein. Da das Gerät aber nicht bekannt ist musst Du das [hier](https://github.com/danielperna84/custom_homematic#homematicip_localexport_device_definition) durchführen.

**Reply by FlorianWicke** [reply]
Und wo geb ich das hier ein?
homematicip_local.export_device_definition

**Reply by SukramJ** [reply]
Entwicklerwerkzeuge -> Dienste, und dass aus der Liste auswählen

**Reply by FlorianWicke** [reply]
Baust du die manuell ein? Ich hätte nämlich noch eines und es kommen evtl. noch 2 dazu?
Datei anhängen oder nur den Inhalt?

**Reply by SukramJ** [reply]
Wie beschrieben als `pull request` bei pydevccu, aber wenn das nicht kannst hier als Dateianhang.

**Reply by FlorianWicke** [reply]
Sorry ich habe es versucht.
Hier der LED-Dimmer.
[LED-Dimmer.zip](https://github.com/danielperna84/hahomematic/files/10435279/LED-Dimmer.zip)

**Reply by SukramJ** [reply]
Kein Problem.
Wenn Du noch mehr Geräte hast, dann am Besten gleich her damit.

**Reply by FlorianWicke** [reply]
Die anderen Geräte muss ich alle erst testen :).
Ich denke aber, da es sich um Dritt-Geräte handelt, dass alle nicht so wirklich erkannt werden ;)

**Reply by SukramJ** [reply]
1. Häng doch bitte Screenshots des Gerätes aus der CCU an.
2. Der Export ist merkwürdig, da die Adressen aus der device_description von denen der paramset_description abweichen. Bitte noch mal wiederholen und hochladen.
3. Bitte beschreibe was man mit dem Gerät machen kann. Ich sehe:
  - Die Kanäle 1-6 sind Taster
  - Die Kanäle 7-14 sind Dimmer (hell/dunkel)
  - In den Kanälen 15 und 16 kann man die Farbe einstellen.

Insgesamt eher ungewöhnlich.

**Comment by FlorianWicke** [comment]
![Bildschirm­foto 2023-01-17 um 16 10 38](https://user-images.githubusercontent.com/56968840/212935399-fd614b51-8238-46d6-8334-6827237dbe5b.png)
![Bildschirm­foto 2023-01-17 um 16 11 21](https://user-images.githubusercontent.com/56968840/212935570-d790a114-4b29-4e18-bdbf-21f8ebef8a90.png)
![Bildschirm­foto 2023-01-17 um 16 11 10](https://user-images.githubusercontent.com/56968840/212935573-16c128c5-6c51-41e8-aa0e-69f7f34466a0.png)



Klar hier.
Das Gerät ist ein 2-fach RGB Dimmer. Es kann aber auch als PWM-Dimmer verwendet werden.
Es hat dann zusätzlich noch 6 Eingänge die digital angesteuert werden können.
Und bei Kanal 15 und 16 kann man RGB-Werte einstellen.

Also des genau was du beschreibst.

**Reply by SukramJ** [reply]
Die Beschreibung reicht mir noch nicht.
Also wenn das ein 2-fach RGB-Dimmer ist, dann gehören die Kanäle 15+16 jeweils zu einem Ausgang.
Welche der Kanäle 7-14 gehören zu dem Ausgang  auf dem die Farbe für 15 geschaltet wird?

Denk bitte noch an einen neuen Export.

**Reply by SukramJ** [reply]
Da ich keine Ahnung hab, ob so ein Gerät noch jemand hat  bzw. mit HA nutzt, und auch noch zu viele Punkte offen sind, würde ich dir erstmal eine Umsetzung als [light template](https://www.home-assistant.io/integrations/light.template/) empfehlen. 
Die dafür notwendigen Entitäten wurden ja alle erzeugt. Damit kannst Du erst mal alles ausprobieren.
Danach könntest Du die erstellten Entitäten hier im Wiki zur Verfügung stellen.

Da dieses Gerät in der Kanalanordnung erheblich von eQ-3 Geräten abweicht würde ich vorerst auf eine Implementierung verzichten.

**Reply by SukramJ** [reply]
Ist [das](http://www.haus-bus.de/LED-Stripe-Dimmer.pdf) das Gerät?

**Comment by FlorianWicke** [comment]
Hi, ja lass es. Ich hab es wieder zurückgegeben. Weil es zu aufwendig ist alle die Geräte einzulernen. :) Sorry für die Störung

**Reply by SukramJ** [reply]
Kein Problem.
